protocol/src/test/scala/io/mediachain/protocol/DataObjectGenerators.scala

```diff
@@ -12,71 +13,134 @@ object DataObjectGenerators {
   import io.mediachain.util.cbor.CborAST._
   import io.mediachain.util.cbor.CValueGenerators._
 
+  val genStringMetas: List[Gen[(String, CValue)]] = (1 to 25).toList.map { _ =>
+    for {
```

Comment:
pretty dang low prio, but someth like this would be cool in the future https://gist.github.com/etorreborre/2231981

applicative style will clean up a looooooooooot of these generators